crates/sdk/src/env/ext.rs

```diff
@@ -147,6 +153,14 @@ impl DraftProposal {
         self
     }
 
+    /// Add an action to delete a proposal.
+    #[must_use]
+    pub fn delete_proposal(mut self, proposal_id: ProposalId) -> Self {
```

Comment:
can we just call it `delete` since the path would already be `Self::external().propose().delete(..)`